src/KristofferStrube.Blazor.Popper/KristofferStrube.Blazor.Popper.csproj

```diff
@@ -8,13 +7,14 @@
     <RepositoryUrl>https://github.com/KristofferStrube/Blazor.Popper</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageVersion>0.4.0</PackageVersion>
-    <TargetFramework>net6.0</TargetFramework>
-    <Authors>Kristoffer Strube</Authors>
+    <PackageVersion>0.5.2</PackageVersion>
+	<TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
+    <Authors>Kristoffer Strube, Mavaddat Javid</Authors>
```

Comment:
Generally only the maintainers or the original project author is included in the Authors list. You would get credited for your contribution through other means.